agent/metasploit_agent.py

```diff
@@ -65,12 +60,18 @@ def process(self, message: m.Message) -> None:
             message: A message containing the path and the content of the file to be processed
 
         """
-        for entry in self.config or []:
+        utils.check_msfrpcd()
+        client = utils.msfrpc_connect()
+        cid = client.consoles.console().cid
+        config = self.args.get("config")
+        if config is None:
+            raise ValueError("Metasploit module(s) must be specified.")
+        for entry in config or []:
```

Comment:
The default to empty list should be done in the intialization of the config variable (self._config=.. or [])